packages/grafana-eslint-rules/rules/no-untranslated-literals.cjs

```diff
@@ -0,0 +1,46 @@
+// @ts-check
+const { ESLintUtils, AST_NODE_TYPES } = require('@typescript-eslint/utils');
+
+const createRule = ESLintUtils.RuleCreator(
+  (name) => `https://github.com/grafana/grafana/blob/main/packages/grafana-eslint-rules/README.md#${name}`
+);
+
+const noUntranslatedLiterals = createRule({
+  create(context) {
+    return {
+      JSXText(node) {
+        const ancestors = context.getAncestors();
+        const isEmpty =  !node.value.trim();
+        const hasTransAncestor = ancestors.some((ancestor) => {
+          return (
+            ancestor.type === AST_NODE_TYPES.JSXElement &&
+            ancestor.openingElement.type === AST_NODE_TYPES.JSXOpeningElement &&
+            ancestor.openingElement.name.type === AST_NODE_TYPES.JSXIdentifier &&
+            ancestor.openingElement.name.name === 'Trans'
+          );
+        });
+        if (!isEmpty && !hasTransAncestor) {
+          context.report({
+            node,
+            messageId: 'noUntranslatedStrings',
+          });
+        }
+      },
+    };
+  },
+  name: 'no-utranslated-strings',
```

Comment:
this name needs to exactly match the `README` subheading else the link won't work correctly. 

think we should be consistent across the board here, either `no-untranslated-strings` or `no-untranslated-literals`.

my vote is for `no-untranslated-strings` as i think it's a little easier to understand 🤔

Reply:
Ah! I will unify all, including the file name, to `no-untranslated-strings` 🤞